share/completions/composer.fish

```diff
@@ -20,12 +20,26 @@ function __fish_composer_using_command
   return 1
 end
 
+function __fish_composer_packages
+    echo "
+import json
+json_data = open('composer.json')
```

Comment:
@krader1961 I totally forgot about this :blush: I'm a bit bummed this last week about some stuff, but I'm going to fix this right now so that I won't forget again!